src/fundamental_interface.jl

```diff
@@ -4,6 +4,11 @@
 #
 ###############################################################################
 
+function Base.:(==)(x::AbstractAlgebra.SetElem, y::AbstractAlgebra.SetElem)
+  x === y && return true
```

Comment:
I've now just added this, to get the tests to run; but this is not meant to preempt any discussion, i.e., we can still *remove* it again, but I wanted to at least know if with this the OscarCI etc. tests complete or (still) don't.